whole-tale/ngx-dashboard
# fix: simplify Docker build (use official base)

## Problem
Docker build for `ngx-dashboard` is complicated and slow.

## Approach
* Change patterns to Docker build/run this image with a focus on performance.

NOTE: this change can go into a later release, as it should not affect runtime performance
NOTE 2: there is a similar PR https://github.com/whole-tale/deploy-dev/pull/50 that will address `make rebuild_dashboard` in `deploy-dev`

## How to Test
1. Run `docker build --pull --no-cache -t wholetale/ngx-dashboard .`
    * The build should complete in a reasonable amount of time (< 5 min)
